upup/models/config/components/kube-apiserver/_do/kube-apiserver.do.options

```diff
@@ -0,0 +1,2 @@
+KubeAPIServer:
+  CloudProvider: external
```

Comment:
We are trying to move this logic to https://github.com/kubernetes/kops/blob/master/pkg/model/components/apiserver.go
This does rely on the `do` tag, which I do see you define later, but soon we hope to have fully moved the tag / options logic to code (it is much easier to understand, IMO)